File: SmileEditor/Model/Legacy/Enemy.cs
namespace SmileEditor.Model.Legacy
{
    internal class Enemy
    {
        internal int Species { get; set; }          // (Legacy) 2 bytes    'pointer (A0) to enemy data
        internal int X { get; set; }                // (Legacy) 2 bytes
        internal int Y { get; set; }                // (Legacy) 2 bytes
        internal int Orientation { get; set; }      // (Legacy) 2 bytes
        internal int Special { get; set; }          // (Legacy) 2 bytes (Prop-X)
        internal int Speed { get; set; }            // (Legacy) 2 bytes
        internal int Unknown3 { get; set; }         // (Legacy) 2 bytes    'graphic unknown
        internal int Unknown4 { get; set; }    		// (Legacy) 2 bytes    'speed2

        internal Enemy() { }
    }
}

File: SmileEditorLegacy/Parsed/Types/Model/Enemy.vb
Public Type Enemy   '16 bytes total
    Species As Integer  '2 bytes    'pointer (A0) to enemy data
    X As Integer    '2 bytes
    Y As Integer    '2 bytes
    Orientation As Integer    '2 bytes
    Special As Integer    '2 bytes (Prop-X)
    Unknown3 As Integer '2 bytes    'graphic unknown
    Speed As Integer    '2 bytes
    Unknown4 As Integer '2 bytes    'speed2
End Type